### PR TITLE
Rest Doc for Full Text Search and withinPolygon

### PR DESCRIPTION
**Pull request by dplewis**
Parse Server has been updated with new features.

See
https://github.com/parse-community/parse-server/pull/3866
https://github.com/parse-community/parse-server/pull/3904

### CODE REVIEW

File: _includes/rest/queries.md [hunk 1]
```diff
@@ -556,6 +557,91 @@ The above example will match any `BarbecueSauce` objects where the value in the
...
+| Parameter           | Use                                                            |
+|--------------------------------------------------------------------------------------|
+| $term               | Specify a field to search (Required)                           |
+| $language           | Determines the list of stop words and the rules for tokenizer. |
+| $caseSensitive      | Enable or disable case sensitive search.     |
+| $diacriticSensitive | Enable or disable diacritic sensitive search |
+
+Please refer to your database documentation on Full Text Search to setup your indexes, weights and limitations.
+
+Note: Postgres doesn't support `$caseSensitive` for Full Text Search, please use $regex above or create a lowercase column in your DB. Postgres supports `$diacriticSensitive: true` by default but `$diacriticSensitive: false` is not supported. To use false automatically, please install Postgres Unaccent Extention and undate your text search configuration.
```
Comment (flovilmart) [inline _includes/rest/queries.md]: Typo: undate

Comment (flovilmart) [reply _includes/rest/queries.md]: Typo: Extention

File: _includes/rest/queries.md [hunk 2]
```diff
@@ -556,6 +557,91 @@ The above example will match any `BarbecueSauce` objects where the value in the
...
+The format `{"$text": {"$search": {parameters}}}`
+
+| Parameter           | Use                                                            |
+|--------------------------------------------------------------------------------------|
+| $term               | Specify a field to search (Required)                           |
+| $language           | Determines the list of stop words and the rules for tokenizer. |
+| $caseSensitive      | Enable or disable case sensitive search.     |
+| $diacriticSensitive | Enable or disable diacritic sensitive search |
+
+Please refer to your database documentation on Full Text Search to setup your indexes, weights and limitations.
```
Comment (flovilmart) [inline _includes/rest/queries.md:573]: Can you put the links to those documentation for mongo 3.2 or 3.4 and PG?

**Review by flovilmart — CHANGES_REQUESTED** [review]
A few typos here and there otherwise it's looking great!
Can you also mention that each feature is available only starting Parse-server 2.5.0?

File: _includes/rest/queries.md (hunk 1 above)
Comment (dplewis) [reply _includes/rest/queries.md]: nice catch

File: _includes/rest/queries.md (hunk 2 above)
Comment (dplewis) [reply _includes/rest/queries.md:573]: Done

**Comment by felipemartim** [comment]
Is this still going to be merged? 👍

**Comment by montymxb** [comment]
@felipemartim yes! 
@flovilmart changes look good to you?

**Review by flovilmart — APPROVED** [review]

**Comment by flovilmart** [comment]
This. Is. Awesome!

**Pull request merged**